addons/circulation/XEH_preInit.sqf

```diff
@@ -73,6 +73,17 @@ PREP_RECOMPILE_END;
     true
 ] call CBA_Settings_fnc_init;
 
+//use aed during compressions - apo_tle
+[
+    QGVAR(AED_duringCpr),
+    "CHECKBOX",
+    LLSTRING(SETTING_AED_duringCpr),
+    [CBA_SETTINGS_CAT, LSTRING(SubCategory_AED)],
+    [false],
```

Comment:
I'd set this to true on default. If you use a "normal" 2 pandle AED placing pads via CPR is common if you know how to do it. Doing analyze requires you to stop tho.